config/locales/en/main.yml

```diff
@@ -2061,6 +2061,9 @@ en:
     change_twilio_auth_token: "Change Auth Token"
     change_api_key: "Change API Key"
     override_code_warning: "Note that the new override code will not work with forms that have already been downloaded. You should record the existing code if this is a problem. Are you sure you want to regenerate the code?"
+    sms_token_warning: "Note that this will invalidate the previous SMS token for this mission. Are you sure you want to regenerate the token?"
```

Comment:
Please insert `All incoming SMSes will be rejected until your SMS gateway settings are updated accordingly.' in the middle.